config/thresholds/mtr-2026.yml

```diff
@@ -22,10 +22,10 @@ pensioner_capital_disregard:
   non_passported: 100_000
   # capital to disregard if monthly disposable income (excluding income derived from capital) is below these figures
   monthly_income_values:
-    [-.inf, 206.99]: 100_000
```

Comment:
My understanding is that the the guidance for these future MTR bands is ambiguous: https://dsdmoj.atlassian.net/browse/MTR-50
So are you simply assuming whole pounds get the higher disregard, simply for consistency with the current rules? Or is there info from somewhere?

Reply:
I was trying to be consistent with current rules, which i think is unnecessary at this point. Reverted the change.